internal/mode/static/nginx/runtime/manager.go

```diff
@@ -143,6 +156,34 @@ func (m *ManagerImpl) Reload(ctx context.Context, configVersion int) error {
 	return nil
 }
 
+// GetUpstreams uses the NGINX Plus API to get the upstreams.
```

Comment:
Is it possible to write tests to cover the lines the linter is complaining about?

Reply:
Maybe I'm reading the coverage report wrong for this file, but I'm pretty sure I'm covering the lines it says I'm not...

Reply:
Oh wait, I might see one thing to fix. But the panic checks should be occurring, so not sure about that.

Reply:
Huh that's weird. It should show up as covered 🤔